plugin/lighthouse/metadata.py

```diff
@@ -566,7 +567,10 @@ def _update_functions(self, fresh_metadata):
             #
 
             if new_metadata.empty:
-                del self.functions[function_address]
+                try:
+                    del self.functions[function_address]
```

Comment:
This may have been to account for a bug I fixed in a55ede77f99d972121d493036b2f5795289f803e, I will see if it is safe to remove now.